locales/README.es_ES.md

```diff
@@ -18,25 +18,25 @@ Rocky Linux es un Sistema Operativo empresarial comunitario diseñado para ser 1
 
 > **P:** ¿A qué te refieres con, "Centos ha cambiado de dirección"?
 
-El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora cambiará a ser un build *upstream* (testeando parches y actualziaciones *antes* de su inclusión en el proveedor upstream).
+El proyecto CentOS [recientemente anunció](https://blog.centos.org/2020/12/future-is-centos-stream/) un cambio en su estrategia para CentOS. Mientras previamente CentOS existía como un build *downstream* de su proveedor upstream (recibía parches y actualizaciones luego del proveedor), ahora pasará a ser un build *upstream* (testeando parches y actualziaciones *antes* de su inclusión en el proveedor upstream).
 
-En adición a eso, el soporte para CentOS Linux 8 se acortó, del 31 de Mayo de 2029 al 31 de Diciembre de 2021.
+Además, el soporte para CentOS Linux 8 se acortó del 31 de Mayo de 2029, al 31 de Diciembre de 2021.
 
 > **P:** ¿Y dónde entra Rocky Linux?
 
-Rocky Linux apunta a funcionar como un build downstream como lo hacía previamente CentOS, compilando releases después de que se agreguen al proveedor upstream, no antes.
+Rocky Linux aspira a funcionar como un build downstream como lo hacía previamente CentOS, compilando releases después de que se agreguen al proveedor upstream, no antes.
```

Comment:
Maybe better "como **una** build". I think is more usual "una build" than "un build"

Reply:
OK